mxgraph-client/pom.xml

```diff
@@ -51,244 +49,198 @@
+                <include>js/view/mxTemporaryCellStates.js</include>
+                <include>js/view/mxCellStatePreview.js</include>
+                <include>js/view/mxConnectionConstraint.js</include>
+                <include>js/handler/mxGraphHandler.js</include>
+                <include>js/handler/mxPanningHandler.js</include>
+                <include>js/handler/mxPopupMenuHandler.js</include>
+                <include>js/handler/mxCellMarker.js</include>
+                <include>js/handler/mxSelectionCellsHandler.js</include>
+                <include>js/handler/mxConnectionHandler.js</include>
+                <include>js/handler/mxConstraintHandler.js</include>
+                <include>js/handler/mxRubberband.js</include>
+                <include>js/handler/mxHandle.js</include>
+                <include>js/handler/mxVertexHandler.js</include>
+                <include>js/handler/mxEdgeHandler.js</include>
+                <include>js/handler/mxElbowEdgeHandler.js</include>
+                <include>js/handler/mxEdgeSegmentHandler.js</include>
+                <include>js/handler/mxKeyHandler.js</include>
+                <include>js/handler/mxTooltipHandler.js</include>
+                <include>js/handler/mxCellTracker.js</include>
+                <include>js/handler/mxCellHighlight.js</include>
+                <include>js/editor/mxDefaultKeyHandler.js</include>
+                <include>js/editor/mxDefaultPopupMenu.js</include>
+                <include>js/editor/mxDefaultToolbar.js</include>
+                <include>js/editor/mxEditor.js</include>
+                <include>js/io/mxCodecRegistry.js</include>
+                <include>js/io/mxCodec.js</include>
+                <include>js/io/mxObjectCodec.js</include>
+                <include>js/io/mxCellCodec.js</include>
+                <include>js/io/mxModelCodec.js</include>
+                <include>js/io/mxRootChangeCodec.js</include>
+                <include>js/io/mxChildChangeCodec.js</include>
+                <include>js/io/mxTerminalChangeCodec.js</include>
+                <include>js/io/mxGenericChangeCodec.js</include>
+                <include>js/io/mxGraphCodec.js</include>
+                <include>js/io/mxGraphViewCodec.js</include>
+                <include>js/io/mxStylesheetCodec.js</include>
+                <include>js/io/mxDefaultKeyHandlerCodec.js</include>
+                <include>js/io/mxDefaultToolbarCodec.js</include>
+                <include>js/io/mxDefaultPopupMenuCodec.js</include>
+                <include>js/io/mxEditorCodec.js</include>
+              </includes>
+              <!-- Skip these since it's not our code. -->
+              <closureStrictModeInput>false</closureStrictModeInput>
+              <closureEmitUseStrict>false</closureEmitUseStrict>
+              <closureWarningLevels>
+                <es5Strict>WARNING</es5Strict>
+              </closureWarningLevels>
```

Comment:
I see these lines are repeated a few times. Could you try to move them in a ``pluginManagement`` section in the root POM? Maybe include also the ``lineSeparator`` (I suppose it's used only for aggregations so it shouldn't affect minification).